### PR TITLE
fix: add version to build flags

### PR DESCRIPTION
**Pull request by Xunzhuo**
Signed-off-by: Xunzhuo <mixdeers@gmail.com>

resolve: https://github.com/mosn/layotto/issues/521

image

### CODE REVIEW

**Comment by Xunzhuo** [comment]
resolve: https://github.com/mosn/layotto/issues/521

File: make/golang.mk [hunk 1]
```diff
@@ -20,6 +20,7 @@ GO_LDFLAGS += -X $(VERSION_PACKAGE).GitVersion=$(VERSION) \
 	-X $(VERSION_PACKAGE).GitCommit=$(GIT_COMMIT) \
 	-X $(VERSION_PACKAGE).GitTreeState=$(GIT_TREE_STATE) \
 	-X $(VERSION_PACKAGE).BuildDate=$(shell date -u +'%Y-%m-%dT%H:%M:%SZ') \
```
Comment (seeflood) [inline make/golang.mk:22]: Seems that these variables are not used? Can we delete them?
$(VERSION_PACKAGE).GitVersion
$(VERSION_PACKAGE).GitCommit
$(VERSION_PACKAGE).GitTreeState
$(VERSION_PACKAGE).BuildDate

**Comment by seeflood** [comment]
@zhenjunMa I checked our previous makefile and found that we build layotto with CGO enabled and some magic scripts:
image
Do you know what they are used for ? 
The new makefile scripts build layotto without CGO and these scripts. Shall we add them?

File: make/golang.mk (hunk 1 above)
Comment (Xunzhuo) [reply make/golang.mk]: Done, I was trying to add more info from build, but it seems like we do not need those additional info.

**Comment by zhenjunMa** [comment]
> @zhenjunMa I checked our previous makefile and found that we build layotto with CGO enabled and some magic scripts: image Do you know what they are used for ? Are they just some redundant arguments copied from MOSN? Currently the new makefile scripts build layotto without CGO and these scripts. Shall we add them?

yes, i just copied from MOSN,  at this time, we can delete these redundant code 😜

**Review by seeflood — APPROVED** [review]
lgtm

**Review by zhenjunMa — APPROVED** [review]

**Pull request merged**